src/components/WritableRemindItem/WritableRemindItem.tsx

```diff
@@ -0,0 +1,157 @@
+    setIsSameMessageChecked(false);
+    setIsSameMessageModalOpen(false);
+  };
+
+  useEffect(() => {
+    if (isFirstRemindItem && isSameMessageChecked) {
+      makeAllRemindMessageSame!();
+    }
+  }, [isFirstRemindItem, isSameMessageChecked, makeAllRemindMessageSame]);
+
+  return (
+    <>
+      <div className="remind-item">
+        <div
+          className="remind-item__header"
+          onClick={handleClickToggleIsItemOpened}>
+          <p className="remind-item__header__title">
+            {remindMonth}월 {remindDay}일에 받을 리마인드 메세지
+          </p>
+
+          <div className="remind-item__header__meta">
+            {isRemindMessageEmpty && (
+              <div
+                className={classNames(
+                  'remind-item__header__warning',
+                  'background-origin-primary',
+                )}>
+                <Icon name="WARNING" size="xs" color="white-100" />
+                <span
+                  className={classNames(
+                    'remind-item__header__warning__text',
+                    'color-origin-white-100',
+                  )}>
+                  리마인드 메세지가 아직 작성되지 않았습니다
+                </span>
+              </div>
+            )}
+            <span className="remind-item__header__button-text">
+              {isRemindMessageEmpty ? '작성하기' : '수정하기'}
+            </span>
+            <Icon
+              name={isItemOpened ? 'ITEM_CLOSE' : 'ITEM_OPEN'}
+              size="xl"
+              color="gray-300"
+              classNameList={['remind-item__header__icon']}
+            />
+          </div>
+        </div>
+        {isItemOpened && (
+          <div
+            className={classNames(
+              'remind-item__message',
+              'background-origin-white-300',
+              {
+                'remind-item__message--open': isItemOpened,
+              },
+            )}>
+            <RemindInput
+              textInput={remindMessage}
+              onChangeInput={handleChangeRemindMessage}
+              placeholder="미래의 내가 받게 될 리마인드 메세지를 작성해보세요 !"
+              maxLength={MAX_REMIND_MESSAGE_LENGTH}
+              editable={true}
+            />
+
+            {isFirstRemindItem && (
+              <span
+                className="remind-item__message__check"
+                onClick={handleClickSameMessageCheck}>
+                <Icon
+                  name={isSameMessageChecked ? 'CHECKED' : 'UN_CHECKED'}
+                  size="2xl"
+                  color="primary"
+                  isFilled={isSameMessageChecked ? true : false}
+                />
+                <p
+                  className={classNames(
+                    'remind-item__message__check__text',
+                    'color-origin-primary',
+                  )}>
+                  항상 같은 리마인드 메세지 받기
+                </p>
+              </span>
+            )}
+          </div>
+        )}
+      </div>
+      {isSameMessageModalOpen && isSameMessageChecked === false && (
```

Comment:
로직이 복잡하네요.. 약간이라도 짧게 쓰려면 이 방법은 어떠신가요
```suggestion
      {isSameMessageModalOpen && !isSameMessageChecked && (
```

Reply:
오오 왜 저렇게 썼을까요 저 ㅋㅋㅋㅋ 감사합니닷